### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by copumpkin** [danlythemanly/xen-blanket#2]
@danlythemanly are you one of the original authors or did you just rescue the repo from google code?

**Comment by danlythemanly** [comment]
Yes I wrote the original version, but have not worked on it in years.

**Comment by copumpkin** [comment]
Ah, that's a pity, but thanks for the original work! Do you have any pointers for what it might mean for a third party to update the patches for recent Xen and other developments? Are there other things that have happened since you stopped that might simplify or complicate this work?

Thanks!

**Comment by danlythemanly** [comment]
The basic idea was to run a PV Xen installation inside an HVM guest.  Since
HVM Xen is supposed to support an unmodified OS, it isn't all that far
off.  The piece that was missing was that the devices exposed to the HVM
guest were PV, and that wasn't something that Xen was able to deal with out
of the box.  So, most of the changes were to interface with devices.  I
don't know how much the paravirtualized device interfaces have changed
since then.

The only other thing to consider is that hypervisor support for nested
virtualization could now be available in some providers (I don't know if it
is or not), which would allow you to just run another hypervisor out of the
box.  You can check cpuinfo on some cloud servers for vmx or equivalent to
see if this is the case.

Good Luck!
Dan
